examples/network/custom_region_demo.py

```diff
@@ -90,8 +90,11 @@ def createNetwork(dataSource):
   # In this demo, we have modified it using sys.path.append since we need it to
   # have an effect on this program.
   sys.path.append(os.path.dirname(os.path.abspath(__file__)))
-  # Add custom region package to network
-  Network.registerRegionPackage("custom_region")
+  
+  custom_region = __import__("custom_region", fromlist=["IdentityRegion"])
```

Comment:
I didn't want to include it at the top incase a user tries to run the example from a different directory since custom_region is only added to the python path in the demo.  I replaced it with a regular import though.